### PR TITLE
Update readme with current `product-details` API

### LINKED ISSUE #86: README might need a refresh
While debugging an issue with release data I tried to find and browse the SVN repo mentioned in the README.

It doesn't appear to exist, but the README implies it does and that it's still used. I _think_ instead the data now comes from https://github.com/mozilla-releng/product-details but we should check this and update the docs accordingly

### PR DESCRIPTION
**Pull request by janbrasna**
Resolves #86

### CODE REVIEW

**Comment by janbrasna** [comment]
@stevejalim I don't think I can do too much about the CI failing here, right? 🤷

**Comment by stevejalim** [comment]
@janbrasna Don't suppose you'd be open to pruning the tox matrix and adding modern Python and Django versions? 🙈

**Comment by janbrasna** [comment]
@stevejalim So while I don't know what I'm doing mostly (shh don't tell anyone) I can tell red cross from green checkmark so I can try… EDIT: so while I can look into supporting different matrix per se, the issue is here: https://github.com/janbrasna/django-product-details/commit/c83d2d32f9bc8ccbbf2c7513f4441b2ae244b8fa ([tests happy](https://github.com/janbrasna/django-product-details/actions/runs/9585377829)) — I can PR that for sure but question is how you'd like that;)

**Comment by janbrasna** [comment]
@stevejalim OK that wasn't that hard: [{py3.9–3.11}-django32|40|42|50 runs](https://github.com/janbrasna/django-product-details/actions/runs/9598251797) I can add to the #90 fix if you'd like. I only won't be able to add 3.12 (migrating away from imp, therefore replacing nose with nose2 or pynose or pytest) that's for someone who knows a thing or two;]

File: README.rst [hunk 1]
```diff
@@ -28,18 +32,20 @@ Mozilla.
 Why?
 ----
 
-The `data source <http://svn.mozilla.org/libs/product-details/>`__ of
-Mozilla Product Details is a PHP library kept on the Mozilla SVN server,
-and was originally written so it could be included into PHP projects via
-an `SVN external <http://svnbook.red-bean.com/en/1.0/ch07s03.html>`__. A
-simple ``svn up`` would fetch the latest data when it became available.
+The data provides by this API is meant to be the source of truth for all
```
Comment (stevejalim) [inline README.rst:35]:
```suggestion
The data provided by this API is meant to be the source of truth for all
```

File: README.rst [hunk 2]
```diff
@@ -28,18 +32,20 @@ Mozilla.
...
-In the meantime, the Product Details library received an additional JSON
-feed, allowing non-PHP projects to consume the data. If, however, the
-consumer is not kept in SVN like the library is, there is no easy way to
-keep the data up to date.
+Historically that data was managed with direct commits into a code
+repository on the Mozilla SVN server. The current version 1.0 of the API
+is meant to be fully compatible with the version previously hosted on
+SVN.
+The `data source <https://github.com/mozilla-releng/product-details/>`__
+of Mozilla Product Details from git repository is published to
```
Comment (stevejalim) [inline README.rst:44]:
```suggestion
of Mozilla Product Details from the git repository is published to
```

File: README.rst [hunk 3]
```diff
@@ -28,18 +32,20 @@ Mozilla.
...
+product-details.mozilla.org.
 
-For Django projects, this app solves that problem.
+For Django projects, this app manages updates and caching of the data,
+and turns the JSONs into Python objects.
```
Comment (stevejalim) [inline README.rst:48]:
```suggestion
and turns the JSON data into Python objects.
```

**Review by stevejalim — APPROVED** [review]
Thanks for this @janbrasna r+wc -- just some small typos to fix

File: README.rst [hunk 4]
```diff
@@ -3,22 +3,26 @@ Mozilla Product Details for Django
...
-    This library holds information about the current builds of Firefox and
-    Thunderbird that Mozilla ships including:
+    Product Details is a public JSON API which contains release information
+    for Firefox (desktop and mobile) and Thunderbird. The data is managed by
+    the Release Management team through Ship-it and exported available at
+    this domain:
 
-    - Latest version numbers for all builds
-    - English and Native names for all languages we support
+    - htttps://product-details.mozilla.org/
```
Comment (stevejalim) [inline README.rst:22]:
```suggestion
    - https://product-details.mozilla.org/
```

Comment (janbrasna) [reply README.rst]: Thanks for reading through it and keeping my Engrish in check;) Most of the content was taken from elsewhere so I'll double check the other sources too bc some things are pretty bizarre;D 

EDIT: yup this one is really [there](https://github.com/mozilla-releng/product-details/blob/03a4f3ada195a1c0caa81bfd0d08e60ade991658/README.rst?plain=1#L5) ;D :shrug:

**Pull request merged**